test-case/check-runtime-pm-status.sh

```diff
@@ -49,62 +52,80 @@ func_check_dsp_status()
 
 func_opt_parse_option $*
 tplg=${OPT_VALUE_lst['t']}
-[[ -z $tplg ]] && dloge "Miss tplg file to run" && exit 1
-
 loop_count=${OPT_VALUE_lst['l']}
+[[ -z $tplg ]] && dloge "Miss tplg file to run" && exit 1
 
 [[ $(sof-dump-status.py --dsp_status 0) == "unsupported" ]] &&
     dlogi "platform doesn't support runtime pm, skip test case" && exit 2
 
 [[ ${OPT_VALUE_lst['s']} -eq 1 ]] && func_lib_start_log_collect
-func_pipeline_export $tplg "type:playback"
+func_pipeline_export $tplg
 func_lib_setup_kernel_last_line
-#TODO: need to go through both playback and capture
+
 for idx in $(seq 0 $(expr $PIPELINE_COUNT - 1))
 do
     channel=$(func_pipeline_parse_value $idx channel)
     rate=$(func_pipeline_parse_value $idx rate)
     fmt=$(func_pipeline_parse_value $idx fmt)
     dev=$(func_pipeline_parse_value $idx dev)
     pcm=$(func_pipeline_parse_value $idx pcm)
+    type=$(func_pipeline_parse_value $idx type)
+
+    case $type in
+        "playback")
+            cmd=aplay
+            dummy_file=/dev/zero
+        ;;
+        "capture")
+            cmd=arecord
+            dummy_file=/dev/null
+        ;;
+        *)
+            dloge "Invalid pipeline: $type (allow value : playback, capture)"
+            exit 1
+        ;;
+    esac
 
```

Comment:
Almost is good to me   
Can you try the bash skill like this usage:  https://github.com/thesofproject/sof-test/blob/master/test-case/multiple-pipeline-playback.sh#L57   

```
help declare
-A        to make NAMEs associative arrays (if supported)
```

so here you can direct to use like: `cmd=${cmd_lst[$type]}`  
if you worry about miss `keyword`, so you can add check for `cmd` is empty  
you can try this skill, I think it is more useful some simple string transform